packages/internal/cli/src/config/env.ts

```diff
@@ -48,6 +48,10 @@ export async function loadVersionEnv() {
   return version;
 }
 
+export const loadNxEnv = () => {
```

Comment:
I don't like this name... It should be sth like `disableNxEnv` for example

Reply:
Hmm, ok. I was rather thinking it's a good place to put other NX-related envs in the future if necessary